expo/gbe/report_views.py

```diff
@@ -165,14 +181,30 @@ def env_stuff(request, conference_choice=None):
 def personal_schedule(request, profile_id='All'):
     viewer_profile = validate_perms(request, 'any', require=True)
 
+    conference_slugs = conf.Conference.all_slugs()
+    if request.GET and request.GET.get('conf_slug'):
+        conference = conf.Conference.by_slug(request.GET['conf_slug'])
+    else:
+        conference = conf.Conference.current_conf()
+
     if profile_id == 'All':
         people = conf.Profile.objects.all().select_related()
     else:
-        people = []
+        people = []  #  Set it to be self, in list format
+
+    tmp_peeps = []
+    for peep in range(0,len(people)):
```

Comment:
If profile_id != "All", then len(people) = 0, and we don't do any of the following and just pass the empty list down into the context. What case does that break?